web-server/v0.4/src/services/dashboard.js

```diff
@@ -117,6 +117,24 @@ export async function queryTocResult(params) {
   return request.post(endpoint);
 }
 
+export async function querySharedConfig(params) {
+  const { id } = params;
+  return request.post('http://localhost:4466/',
```

Comment:
We should reference the `graphql` key from our datastoreConfig field within the Redux store in order to make requests against the production GraphQL service.